test/appium/tests/test_wallet.py

```diff
@@ -9,14 +9,45 @@ class TestWallet(SingleDeviceTestCase):
 
     @pytest.mark.wallet
     def test_wallet_error_messages(self):
+        sender = transaction_users_wallet['A_USER']
+        recipient = transaction_users_wallet['B_USER']
         sign_in_view = SignInView(self.driver)
-        sign_in_view.create_user()
-        wallet_view = sign_in_view.wallet_button.click()
+        sign_in_view.recover_access(sender['passphrase'], sender['password'])
+        home_view = sign_in_view.get_home_view()
+        wallet_view = home_view.wallet_button.click()
         send_transaction = wallet_view.send_button.click()
+
+        # Check valid amount
         send_transaction.amount_edit_box.send_keys('asd')
         send_transaction.find_full_text('Amount is not a valid number')
-        send_transaction.amount_edit_box.send_keys('0,1')
+        send_transaction.amount_edit_box.clear()
+
+        # Check insufficient funds
+        send_transaction.amount_edit_box.send_keys('100000')
         send_transaction.find_full_text('Insufficient funds')
+        send_transaction.amount_edit_box.clear()
+
+        # Check invalid address
+        incorrect_public_key = '5261ceba31e3a7204b498b2dd20220a6057738d'
+        send_transaction.chose_recipient_button.click()
+        send_transaction.enter_recipient_address_button.click()
+        send_transaction.enter_recipient_address_input.set_value(incorrect_public_key)
+        send_transaction.done_button.click()
+        send_transaction.error_dialog.wait_for_error_message('Invalid address')
+        send_transaction.error_dialog.ok_button.click()
+
+        # Check wrong password
+        incorrect_password = 'wrongpasswd'
+        send_transaction.chose_recipient_button.click()
+        send_transaction.enter_recipient_address_button.click()
+        send_transaction.enter_recipient_address_input.set_value(recipient['address'])
+        send_transaction.done_button.click()
+        send_transaction.amount_edit_box.click()
+        send_transaction.amount_edit_box.set_value('0.00001')
+        send_transaction.sign_transaction_button.click()
+        send_transaction.enter_password_input.send_keys(incorrect_password)
+        send_transaction.sign_transaction_button.click()
+        send_transaction.find_full_text('Wrong password')
```

Comment:
Let's either append verification errors to a list and check it in the end of the test or split it

Reply:
Ok, let's use [self.verify_no_errors()](https://github.com/status-im/status-react/blob/develop/test/appium/tests/base_test_case.py#L104).
You can see examples [in messaging tests](https://github.com/status-im/status-react/blob/develop/test/appium/tests/test_messaging.py)